lib/puppet/provider/certs_bootstrap_rpm/katello_ssl_tool.rb

```diff
@@ -33,7 +33,17 @@ def run
   protected
 
   def last_rpm
-    Dir.glob(File.join(resource[:dir], "#{resource[:name]}-*.noarch.rpm")).sort.last
+    rpms = Dir.glob(File.join(resource[:dir], "#{resource[:name]}-*.noarch.rpm"))
+
+    rpms = rpms.collect do |rpm|
+      rpm_split = rpm.split("#{resource[:name]}-")[1].split('.noarch.rpm')[0]
+      version = rpm_split.split('-')[0]
+      release = rpm_split.split('-')[1]
+
+      {'release' => release, 'rpm' => rpm}
+    end
+
+    rpms.sort { |a,b| a['release'].to_i <=> b['release'].to_i }.last
```

Comment:
AFAIK we don't increment the version (https://github.com/Katello/puppet-certs/pull/83/files/e6df6f10f485e0bf283c428c371fa2606b7faaa5#diff-d7c04bc14bf477574cb5d1f4d285675cR13) only the release when we generate a new bootstrap RPM. So possible that logic could change down the road, but I don't forsee it :)